inc/webfont_manager.php

```diff
@@ -20,13 +20,15 @@ static function single() {
 		return $single;
 	}
 
-	function add_font( $name, $weights = array() ) {
+	function add_font( $name, $weights = array(), $subset = 'latin' ) {
 		if( empty( $this->fonts[$name] ) ) {
-			$this->fonts[$name] = $weights;
-		}
-		else {
-			$this->fonts[$name] = array_merge( $this->fonts[$name], $weights );
-			$this->fonts[$name] = array_unique( $this->fonts[$name] );
+			$this->fonts[ $name ] = array(
+				'variants' => $weights,
+				'subset' => $subset,
+			);
+		} else {
+			$this->fonts[ $name ] = array_merge( $this->fonts[$name], $weights );
+			$this->fonts[ $name ] = array_unique( $this->fonts[$name] );
```

Comment:
```suggestion
			$this->fonts[ $name ] = array_merge( $this->fonts[$name], $weights );

			if ( is_array( $this->fonts[$name] ) ) {
				$this->fonts[$name] = array_unique( $this->fonts[$name] );
			}
```

@Misplon  Try changing the highlighted lines to this. Does that prevent the notice from appearing, and working as intended?

Reply:
Assuming this change is correct then, unfortunately, this hasn't helped.

<img width="1169" alt="settings" src="https://user-images.githubusercontent.com/789159/113280720-f5dc8200-92e4-11eb-8388-28dfdeabdfa3.png">

Reply:
Yeah, that's correct. :(
I've set up a fresh install and I'm not able to replicate this error on that install either - I'm using Corp there also. Are you able to replicate this on another install?